ecosystem/typescript/sdk_v2/src/transactions/authenticator/transaction.ts

```diff
@@ -181,3 +182,35 @@ export class TransactionAuthenticatorFeePayer extends TransactionAuthenticator {
     return new TransactionAuthenticatorFeePayer(sender, secondary_signer_addresses, secondary_signers, fee_payer);
   }
 }
+
+export class TransactionAuthenticatorSecp256k1 extends TransactionAuthenticator {
```

Comment:
I believe we also need to add it to the main `TransactionAuthenticator` class https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk_v2/src/transactions/authenticator/transaction.ts#L13

Reply:
Yep fixed